### PR TITLE
Add node.js integration tests

### PR DESCRIPTION
**Pull request by stringhandler**
## Description

Adds a simple integration test that uses GRPC to add a block and check that the chain height changes

## Motivation and Context

The idea here is to start some integration tests that can be added over time

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [ ] I'm merging against the `development` branch.
* [ ] I ran `cargo-fmt --all` before pushing.
* [ ] I ran `cargo test` successfully before submitting my PR.
* [ ] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: integration_tests/test/base_node.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        defaults: true,
+        oneofs: true
+    });
+const protoDescriptor2 = grpc.loadPackageDefinition(packageDefinition2);
+const tariWallet = protoDescriptor2.tari.rpc;
+walletClient = new tariWallet.Wallet('127.0.0.1:50061', grpc.credentials.createInsecure());
+grpc_promise.promisifyAll(walletClient);
+
+describe('Base Node', function () {
+    this.timeout(10000); // five minutes
```
Comment (stringhandler) [inline integration_tests/test/base_node.js:39]: Lol

Comment (delta1) [reply integration_tests/test/base_node.js:39]: lmao

**Comment by delta1** [comment]
`cargo fmt` fail - did you have to switch off "format on save" because of JS? I feel that pain.

**Comment by stringhandler** [comment]
Nope, was looking for feedback, then will run it

**Comment by delta1** [comment]
Nice seems to be working

**Review by delta1 — APPROVED** [review]
Needs some docs but a good start

**Review by sdbondi — APPROVED** [review]
Nice!

**Pull request merged**